api/v1beta2/cluster_phase_types.go

```diff
@@ -47,10 +47,6 @@ const (
 	// but its infrastructure has not yet been fully deleted.
```

Comment:
A few places to update:
* healthchecking.md: "- If a Machine fails for any reason (if the FailureReason is set), the Machine will be remediated immediately"
* Maybe? bootstrap-provider.plantuml & machine-infra-provider.plantuml